pkg/crc/gpg/gpg.go

```diff
@@ -45,7 +45,7 @@ func GetVerifiedClearsignedMsgV3(pubkey, clearSignedMsg string) (string, error)
 	}
 	block, rest := goClearsign.Decode([]byte(clearSignedMsg))
 	if len(rest) != 0 {
-		return "", fmt.Errorf("Error decoding clear signed message: %w", err)
+		return "", fmt.Errorf("Error decoding clear signed message")
```

Comment:
Bit confusing commit log, what we were doing is wrapping a `nil` error to return error object which is wrong.

Reply:
What matters to me is that the error is unrelated to the current function call (`goClearsign.Decode`), maybe it's set, maybe not, but its value does not matter to me, it's irrelevant to the current context.

Reply:
In commit log `the err variable is related to the  goClearsign.Decode call` => `the err variable is related to the  goOpenpgp.ReadArmoredKeyRing call` ?

Reply:
Thanks, I reworked the log once more ^^ :)